hyperspy/tests/signals/test_find_peaks1D_ohaver.py

```diff
@@ -37,10 +39,14 @@ def setup_method(self, method):
 
     def test_find_peaks1D_ohaver_high_amp_thres(self):
         signal1D = self.signal
-        peak_list = signal1D.find_peaks1D_ohaver(amp_thresh=10.0)[0]
+        peak_list = signal1D.find_peaks(amp_thresh=10.0).data[0]
+        if isinstance(peak_list, da.Array):
+            peak_list = peak_list.compute()
         assert len(peak_list) == 0
 
     def test_find_peaks1D_ohaver_zero_value_bug(self):
         signal1D = self.signal
-        peak_list = signal1D.find_peaks1D_ohaver()[0]
+        peak_list = signal1D.find_peaks().data[0]
```

Comment:
```suggestion
        peak_list = signal1D.find_peaks().data[1]
```

Reply:
For our future self and reader, @CSSFrancis's suggestion was for demonstrating GitHub usage in a tutorial and it is random! 😅